1table.sql

```diff
@@ -0,0 +1,38 @@
+use RecruitingEducatorsCompanyDB
+go
+
+drop table if exists RecruitingEducatorsCompany
+go
+
+create table dbo.RecruitingEducatorsCompany(
+EducatorsId int not null identity primary key,
+FirstName Varchar(30) not null 
+    constraint ck_Recruiting_firstname_cannot_be_blank check(Firstname <> ''),
+LastName varchar(30) not null
+    constraint ck_Recruiting_lastname_cannot_be_blank check(lastname <> ''),
+DOB date not null
```

Comment:
No need for constraint on DOB. You are restricting it to be before DateContacted.